ddln_imaginator.proto

```diff
@@ -185,7 +186,45 @@ message WorldInfo {
     optional LatLongAlt home_geo = 4;
 }
 
+// Imaginator info message which contains imaginator specific information
+// This information is world agnostic and typically generated upon initialization
+message ImaginatorInfo {
+
+    // Reserved for future values such as version info and so on.
+    reserved 1, 2, 3, 4, 5, 6, 7, 8, 9;
```

Comment:
no need to specify these, just use 10 below if you wish
also we can use 1 and use 2... later